process/src/main/kotlin/com/alfresco/process/models/Fields.kt

```diff
@@ -38,6 +38,7 @@ data class Fields(
     @Json(name = "enableFractions") @field:Json(name = "enableFractions") var enableFractions: Boolean? = null,
     @Json(name = "enablePeriodSeparator") @field:Json(name = "enablePeriodSeparator") var enablePeriodSeparator: Boolean? = null,
     @Json(name = "currency") @field:Json(name = "currency") var currency: String? = null,
+    @Json(name = "dateDisplayFormat") @field:Json(name = "dateDisplayFormat") var dateDisplayFormat: String? = null,
```

Comment:
Why did you assign as null if you declare as optional?

Reply:
If there is a custom date format then it will return as String otherwise null. If I do not set the default value as null because   "dateDisplayFormat" key only included in the response if type is "datetime"